sdk/python/kfp/dsl/__init__.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 from kfp.components.component_decorator import component
+from kfp.components.container_component_decorator import container_component
```

Comment:
What do you think about also putting `kfp.components.structures.ContainerSpec` here?

In general, we don't consider the `structures.py` module to be part of the public/stable API. We change the data structures in that module readily, without much consideration for backward compatibility.

Putting `kfp.components.structures.ContainerSpec` under the `dsl` module, amounts to saying that we consider it to be part of the public/stable API.